File: app/src/main/java/net/imatruck/dsmanager/models/TaskAdditionalTracker.java
package net.imatruck.dsmanager.models;

import com.google.gson.annotations.SerializedName;


@SuppressWarnings("unused")
public class TaskAdditionalTracker {

    private String url;
    private String status;
    @SerializedName("update_timer")
    private int updateTimer;
    private int seeds;
    private int peers;

    public String getUrl() {
        return url;
    }

    public String getStatus() {
        return status;
    }

    public int getUpdateTimer() {
        return updateTimer;
    }

    public int getSeeds() {
        return seeds;
    }

    public int getPeers() {
        return peers;
    }
}

File: app/src/main/java/net/imatruck/dsmanager/models/TaskAdditionalTracker.kt
package net.imatruck.dsmanager.models

import com.google.gson.annotations.SerializedName


class TaskAdditionalTracker {
    val url: String? = null
    val status: String? = null
    @SerializedName("update_timer")
    val updateTimer: Int = 0
    val seeds: Int = 0
    val peers: Int = 0
}
